docs/DeveloperGuide.md

```diff
@@ -249,14 +249,29 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a…                    | I can…                                                   | So that I can…                                                                                          |
+| -------- | ------------------------ | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| * *      | Meticulous OHS admin     | have automatic backups of my data                        | rest knowing my data will not be accidentally erased.                                                   |
+| * *      | OHS admin                | generate audit reports for financial data                | comply with internal audits of the Residential College.                                                 |
+| * *      | OHS admin                | export records to csv files                              | easily create mailing lists or send relevant data to other admin.                                       |
+| * * *    | New/Confused User        | check the syntax for a command                           | do a task even if I am unsure of the command usage.                                                     |
+| * * *    | First Time User          | ask for help                                             | quickly and easily learn how to use the application in one place.                                       |
+| * * *    | OHS Admin                | view a list of all students                              | check which students are in the system and access their particulars.                                    |
+| * * *    | OHS Admin                | so that I can check which students stay in which rooms.  | check which students are in the system and access their particulars.                                    |
+| * *      | As as skeptical GUI user | create aliases to other commands                         | perform my common actions while typing less.                                                            |
+| * *      | skeptical GUI user       | create compound commands                                 | ... e.g. to view bills and pay them at the same time, perform common sequences of actions more quickly. |
```

Comment:
Should be removed.

Reply:
I'll remove the redundant line. The rest shouldn't be removed IMO - they are stories we've come up with.

Reply:
Yep my bad! Was referring to the `create compound commands` user story only.